packages/react-conformance/src/defaultTests.tsx

```diff
@@ -450,10 +450,10 @@ export const defaultTests: DefaultTestObject = {
       const { testOptions = {} } = testInfo;
 
       const propNames = Object.keys(componentInfo.props);
-      const ignoreProps = testOptions['consistent-callback-args']?.ignoreProps || [];
+      const legacyCallbacks = testOptions['consistent-callback-args']?.legacyCallbacks || [];
```

Comment:
Might want to add a check that every entry in `legacyCallbacks` also exists in `propNames` (i.e. `legacyCallbacks` doesn't contain anything that's not a prop). Otherwise a typo or refactoring could break this test.

Reply:
Good point! I added a check after this line